packages/components/src/Combobox/components/ComboboxActivator/ComboboxActivator.tsx

```diff
@@ -1,24 +1,20 @@
 import React from "react";
-import { Button } from "@jobber/components/Button";
-import { Chip } from "@jobber/components/Chip";
 import { ComboboxContext } from "../../ComboboxProvider";
 import { ComboboxActivatorProps } from "../../Combobox.types";
+import { useComboboxActivatorAccessibility } from "../../hooks/useComboboxActivatorAccessibility";
 
 export function ComboboxActivator(props: ComboboxActivatorProps) {
-  const { handleClose, open, setOpen } = React.useContext(ComboboxContext);
+  const { handleClose, open, handleOpen } = React.useContext(ComboboxContext);
+  const { htmlAttributes } = useComboboxActivatorAccessibility();
 
-  if (props.children.type === Button || props.children.type === Chip) {
-    return React.cloneElement(props.children, {
-      role: "combobox",
-      onClick: () => {
-        if (open) {
-          handleClose();
-        } else {
-          setOpen(true);
-        }
-      },
-    });
-  }
-
-  return props.children;
+  return React.cloneElement(props.children, {
+    ...htmlAttributes,
```

Comment:
this puts us in a funny state

if you give us a `div` for a custom activator it will actually have better accessibility since we're adding aria-controls and aria-expanded

![image](https://github.com/user-attachments/assets/56f59020-d234-4b39-ac5c-c727a8721172)


however providing a Chip, or Button - we won't apply those 😬 which isn't great since our own internal `ComboboxTrigger` is a Chip

![image](https://github.com/user-attachments/assets/6d663293-cf34-4685-a748-930ba28c59fc)

this is because Chip and Button don't accept an `ariaControls` nor `ariaExpanded`. we'd need to open it up to either those specific values, or aria stuff in general